### PR TITLE
Use fluent icons new codepoints for datepicker, timepicker, flipview, scrollbar, pipspage

### PR DESCRIPTION
**Pull request by licanhua**
1. Use fluent icons new codepoints for datepicker, timepicker, flipview, scrollbar, pipspage
2. Add RequireInsiderSDK to support 21h1 fonts to compiled even if there is no 21h1 insderSDK

CalendarView:
image

LoopSelector for datepicker and timepicker:
image

image

FlipView
image
image

Scrollbar:
image
image
image
image

PipsPager
image
image

### CODE REVIEW

File: dev/PipsPager/PipsPager_21h1_themeresources.xaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <x:String x:Key="PipsPagerPreviousPageButtonGlyph">&#xF08D;</x:String>
```
Comment (marksfoster) [inline dev/PipsPager/PipsPager_21h1_themeresources.xaml:6]: &#xF090;

File: dev/PipsPager/PipsPager_21h1_themeresources.xaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    <x:String x:Key="PipsPagerNextPageButtonGlyph">&#xF08F;</x:String>
```
Comment (marksfoster) [inline dev/PipsPager/PipsPager_21h1_themeresources.xaml:7]: &#xF08E;

<x:String x:Key="PipsPagerNextPageButtonGlyph">&#xF08E;</x:String>

Comment (licanhua) [reply dev/PipsPager/PipsPager_21h1_themeresources.xaml:7]: You are right. I just notice it when I do the screenshot on it

**Review by ranjeshj — APPROVED** [review]
:shipit:

**Comment by mdtauk** [comment]
Why the move away from Chevrons to Caret?  The FluentUI Icons include both.

image

image

**Pull request merged**